core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -52,13 +60,14 @@ export class DotEditContentFormComponent implements OnInit {
      */
     initilizeForm() {
         this.form = this.fb.group({});
-        this.formData.layout.forEach(({ columns }) => {
-            columns?.forEach((column) => {
-                column.fields.forEach((field) => {
-                    const fieldControl = this.initializeFormControl(field);
-                    this.form.addControl(field.variable, fieldControl);
-                });
-            });
+
+        this.formData.fields.forEach((field) => {
```

Comment:
I was going to say: "filter first" and you will say "NO because I don't want to run through the array twice, you old men"

Reply:
You know me well.